OKR/JS_CORE.md

```diff
@@ -2,6 +2,27 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2019 Q1
+
+### A complete implementation of IPFS in JavaScript
+
+* p0 - Base32 encoded version 1 CIDs are the default for new content added to IPFS [@alanshaw](https://github.com/alanshaw)
+* p0 - IPNS has a streaming API [@vasco-santos](https://github.com/vasco-santos)
+* p1 - IPNS supports `ttl`, `dhtt` and `dhtrc` parameters and republish configurations [@vasco-santos](https://github.com/vasco-santos)
+* p1 - A repo migration tool exists and JS IPFS can automatically upgrade older repos [@achingbrain](https://github.com/achingbrain)
+* p2 - Vital missing APIs are implemented i.e. `/dag/put`, `/dag/resolve`, `/repo/gc`, `/refs` and `/swarm/filters/*` [@achingbrain](https://github.com/achingbrain)
+
+### Used in production
+
+* p0 - A JS IPFS daemon is one (or more!) of the IPFS gateway nodes [@alanshaw](https://github.com/alanshaw)
+* p2 - Perf benchmarking site is launched and 10 different benchmarking scenarios exist [@alanshaw](https://github.com/alanshaw)
+* p0 - IPFS is a transport for [npm/tink](https://github.com/npm/tink) [@achingbrain](https://github.com/achingbrain)
+
+### Inviting for developers and contributors
+
+* p1 - Continuous deployment drives `js-ipfs-unixfs-*` and `js-ipfs-mfs` [@achingbrain](https://github.com/achingbrain)
```

Comment:
What work does this KR translate into?

Reply:
Once we settle on a new CI this'll involve:

1. Figuring out a strategy to use e.g. solving questions like:
    * does publish a prerelease or a release?
    * how does CI know what version number to increment?
    * when does a release happen? push to master?
1. Configuring the CI to enact the release ensuring secrets are handled correctly
1. Testing the setup beforehand on a test module